eras/alonzo/test-suite/src/Test/Cardano/Ledger/Alonzo/AlonzoEraGen.hs

```diff
@@ -486,7 +487,7 @@ sumCollateral ::
   UTxO era ->
   Coin
 sumCollateral tx (UTxO utxo) =
-  coin . balance @era . UTxO . eval $ collateral_ ◁ utxo
+  coin . balance @era . UTxO $ SplitMap.restrictKeysSet utxo collateral_
```

Comment:
Perhaps like this?

coin . balance @era . UTxO  $ (collateral_ Split.◁  utxo)

Reply:
Same here, I don't think we need to try and use same unicode operator in the test suite.